.github/pull_request_template.md

```diff
@@ -24,6 +24,7 @@ Please answer these questions before creating your pull request. Thanks!
    - [ ] I am adding new credentials
    - [ ] I am adding a new dependency
    - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.
+   - [ ] I acknowledge that I have ensured my changes to be thread-safe. Follow the link for more information: https://docs.google.com/document/d/162d_i4zZ2AfcGRXojj0jByt8EUq-DrSHPPnTa4QvwbA/edit#bookmark=id.gotyiyvro5mr
```

Comment:
In the future, are we going to add this to our document? In that we, we should log a Jira to change this link

Reply:
Another thing is that this document is only accessible in snowflake. Since we are a public API, should we make a public version of this document?

Reply:
@sfc-gh-yuwang made a good point, there could be external contributors. we'd better have this as part of our contribution guidance

Reply:
@sfc-gh-yuwang I will move this to `CONTRIBUTOR.md` but will do it later. https://snowflakecomputing.atlassian.net/browse/SNOW-1673185